src/main/java/org/springframework/integration/dsl/IntegrationFlowDefinition.java

```diff
@@ -2346,7 +2347,7 @@ public void accept(GenericEndpointSpec<BridgeHandler> bridge) {
 			}
 		}
 		if (hasSubFlows) {
-			channel(afterRouterChannel);
+			handle(bridgeHandler);
```

Comment:
Can we not avoid the bridge for cases where there __is__ continuing flow after the router?